<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainQMW</class>
 <widget class="QMainWindow" name="MainQMW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>КОТИК ЛР № 1 з. 4-5</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Открыть файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="fileOpenProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fileProgressLabel">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Обработка файла может занимать некоторое время
Ждите</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kotikFormatConvertButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Перевод в формат .kotik</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sourceFormatButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Перевод в исходный формат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="kotikTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>231</width>
          <height>311</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>341</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="fileHexContentTextEdit">
      <property name="minimumSize">
       <size>
        <width>661</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
